apps/mobile/src/hooks/use-account-identifier.ts

```diff
@@ -0,0 +1,129 @@
+import { useMemo } from 'react';
+
+import { useAccountsByFingerprint } from '@/store/accounts/accounts.read';
+import { useBitcoinAccounts } from '@/store/keychains/bitcoin/bitcoin-keychains.read';
+import {
+  useStacksSignerAddressFromAccountIndex,
+  useStacksSigners,
+} from '@/store/keychains/stacks/stacks-keychains.read';
+import { useWallets } from '@/store/wallets/wallets.read';
+
+import { inferPaymentTypeFromPath } from '@leather.io/bitcoin';
+import { extractAccountIndexFromPath } from '@leather.io/crypto';
+import { AccountIdentifier } from '@leather.io/models';
+import { isDefined, uniqueArray } from '@leather.io/utils';
+
+function getDescriptorFromKeychain<T extends { keyOrigin: string; xpub: string }>(
```

Comment:
Maybe `getDescriptorFromKeychain` could move into `@leather.io/bitcoin` ?

Reply:
+1 would be great to keep the pure methods located in the domain packages